torch/lib/c10d/comm.hpp

```diff
@@ -49,6 +49,9 @@ class GradBucket {
     return tensors_;
   }
 
+  // Each tensor in the list that getPerParameterTensors corresponds to a parameter.
+  std::vector<at::Tensor> getPerParameterTensors() const;
```

Comment:
Could we add python unittests that ensure `getPerParameter()` succeeds, i.e. compare it to the tensor for a given parameter in python model?

Reply:
I don't think it's necessary since the caller powerSGD_hook is involved in the unit test, and I have also run PowerSGD e2e to make sure no accuracy/performance regression.